spec/supports/helpers.rb

```diff
@@ -0,0 +1,20 @@
+require 'aws-sdk'
+
+def purge_queue
+  # Ideally we would just call `@sqs.purge_queue(:queue_url => @queue_url)`,
+  # but this method can only be called once every 60 seconds.
+  receive_all_messages
+end
+
+def receive_all_messages
+  messages = []
+  poller = Aws::SQS::QueuePoller.new(@queue_url)
```

Comment:
I *think* when you define a value using `let` as on line line [16 of the spec](https://github.com/logstash-plugins/logstash-output-sqs/blob/68d400fe926b1e71bedeee0b7b3a205631ee7f97/spec/unit/outputs/sqs_spec.rb#L16) it creates a local variable (identifier by the `@`) in the spec class.

Since the `receive_all_messages` is then created in the same scope as the local variable, it can access it using the `@`.